src/css/components/_footer.scss

```diff
@@ -0,0 +1,3 @@
+footer {
+  border-top: 1px solid #021e37;
```

Comment:
I'm not sure how this is done in tailwind, but I think there's a [variable](https://github.com/luckyframework/website/blob/master/tailwind.config.js#L131) for this? If possible, can you use that?

Reply:
all set.